spec/factories/books.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
```

Comment:
the name of the file should be singular, like the factory. You could also name the file `book_factory.rb` if you want, to avoid confusion when you are searching for a specific file